frontends/concrete-python/Makefile

```diff
@@ -145,6 +145,9 @@ test-notebooks:
 	eval $(shell make silent_cp_activate)
 	./scripts/jupyter/jupyter.sh
 
+test-notebooks-macos:
```

Comment:
there was another issue on mac (doesn't find ./scripts/jupyter/jupyter.sh), I hope it's gonna fix the issue